ui/src/containers/AppsTableContainer.js

```diff
@@ -40,7 +38,7 @@ export const AppsTableContainer = ({
 
   useEffect(() => {
     getApps();
-  }, []);
+  }, [getApps]);
```

Comment:
I saw this before as a missing dependency - are there any side-effects of adding it here do you know?

Reply:
Here is the warning message:

```
React Hook useEffect has a missing dependency: 'getApps'. Either include it or remove the dependency array.
```

It seems to indicate that an empty array is not the right thing to do. It doesn't seem to have any side effect.